packages/cli/src/commands/migrate.ts

```diff
@@ -257,6 +250,29 @@ migrateCommand
             }
           },
         },
+        {
+          title: 'Checking for TypeScript errors',
+          enabled: (ctx): boolean => !ctx.skip,
+          task: async (_ctx, task) => {
+            const moduleManager = await getModuleManager();
+            try {
+              await runModuleCommand(['tsc'], { cwd: options.basePath });
```

Comment:
Just a note, but we're going to need to figure out how we select between `tsc` and using `glint` here. Probably need to plumb it through `ctx` in the initialization step.

Reply:
@brendaHuang ^^^^